src/components/dashboard/activity-feed/utils.tsx

```
@@ -0,0 +1,69 @@
+            }
+        case 'IssuesEvent':
+            return {
+                icon: <Bug className="w-4 h-4 text-red-600" />,
+                actionText: `${event.payload.action} issue`
+            }
+        case 'ForkEvent':
+            return { icon: <GitFork className="w-4 h-4 text-orange-600" />, actionText: 'forked repository' }
+        case 'WatchEvent':
+            return { icon: <Star className="w-4 h-4 text-yellow-600" />, actionText: 'starred repository' }
+        case 'PushEvent':
+            const commits = event.payload.size || 0
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)